rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Please kindly add `Getters` and `Setters` for `@book` and `@person` instance variables so as to be able to modify them from the `Rental` class like so `attr_accessor  :book, :person` .